core/shared/src/test/scala/com/monovore/decline/HelpSpec.scala

```diff
@@ -42,20 +42,28 @@ class HelpSpec extends AnyWordSpec with Matchers {
         (first, second, third, flagOpt, flagOpts, subcommands).tupled
       }
 
+      println(Help.fromCommand(parser).render(HelpFormat.Colors))
+
       Help.fromCommand(parser).toString should equal(
-        """Usage: program [--first] [--second <integer>] [--third <integer>] [--flagOpt[=<string>]] --flag[=<string>] [--flag[=<string>]]... run
+        """
+          |Usage:
+          |    program [--first] [--second <integer>] [--third <integer>] [--flagOpt[=<string>]] --flag[=<string>] [--flag[=<string>]]... run
           |
           |A header.
           |
           |Options and flags:
           |    --first, -F
           |        First option.
+          |
```

Comment:
Can we do this without updating the existing spacing?

Reply:
Yep, I just always felt the default is a bit too crowded so thought I'd sneak it in :P Will revert.

Reply:
Yeah... I don't know if I disagree on an aesthetic level, but there are a lot of benefits to compactness in this context. (This change on its own makes this section ~50% larger, and now options get more spacing than section headers, which would probably want even more breathing room proportionately...)